homeassistant/components/duckdns/__init__.py

```diff
@@ -42,25 +46,30 @@
     token = config[DOMAIN][CONF_ACCESS_TOKEN]
     session = async_get_clientsession(hass)
 
-    result = await _update_duckdns(session, domain, token)
-
-    if not result:
-        return False
-
-    async def update_domain_interval(now):
+    async def update_domain_interval(_now):
         """Update the DuckDNS entry."""
-        await _update_duckdns(session, domain, token)
+        return await _update_duckdns(session, domain, token)
+
+    intervals = (
+        INTERVAL,
```

Comment:
I think that initially we should retry within a minute.

Reply:
1min, 5min, 15min, and then every 30min?

Reply:
sure

Reply:
First item is the normal interval, new intervals below